src/main/java/com/google/firebase/messaging/FirebaseMessaging.java

```diff
@@ -91,7 +95,7 @@ public String send(@NonNull Message message) throws FirebaseMessagingException {
    *
    * <p>If the {@code dryRun} option is set to true, the message will not be actually sent. Instead
    * FCM performs all the necessary validations and emulates the send operation. The {@code dryRun}
-   * option is useful for determining whether an FCM registration has been deleted. However, it 
+   * option is useful for determining whether an FCM registration has been deleted. However, it
```

Comment:
Would "FCM registration token" be more accurate?

If so, change here and throughout.

Reply:
Similar to the `@throws  FirebaseMessagingException`. I'm following the comment for `send`: https://github.com/firebase/firebase-admin-java/blob/b90979e10cb1f62c1bf6741bb06848c96d3ec17a/src/main/java/com/google/firebase/messaging/FirebaseMessaging.java#L98
I'm happy to make changes to all the places to keep them consistent if needed.

Reply:
Thanks Kevin! Marking this as resolved.